package.json

```diff
@@ -32,6 +32,7 @@
     "prepare:style": "georender-style2png node_modules/georender-style2png/example/style.json -o public/style.png",
     "prepare:wasm": "cp node_modules/eyros/2d.wasm public/eyros2d.wasm",
     "browserify": "browserify -p tinyify app.js | indexhtmlify --title 'peermaps' > public/index.html",
+    "test:browserify": "browserify -p tinyify app.js",
```

Comment:
I suspect piping into `indexhtmlify` doesn't cause a proper process error, so removing this for now, since I just want to test the browserify step by itself first. Also, if this fails, the build must fail.